encoders/image/PaddleHubEncoder/requirements.txt

```diff
@@ -0,0 +1 @@
+jina
```

Comment:
no other dependency is required?

Reply:
paddlehub is required as well, updated the PR.

Reply:
does the docker build -f Dockerfile .... work?